databases/functions/userFollowUser/user-block.js

```diff
@@ -41,7 +42,7 @@ module.exports = async (
     user_id_follower: targetUserId,
     user_id_followed: selfUserId,
     action: 'out',
-    source: `postId:${postId}`
+    source: commentId ? `commentId:${commentId}` : `postId:${postId}`
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The source field is now dynamic and can be either `commentId` or `postId`. This change could potentially affect other parts of the codebase that rely on the `source` field. Ensure that all dependent code is updated to handle this new format.
```diff
- source: commentId ? `commentId:${commentId}` : `postId:${postId}`
+ source: commentId ? `commentId:${commentId}` : `postId:${postId}` // ensure all dependent code is updated to handle this new format
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->